src/bci_build/package/spack/README.md.j2

```diff
@@ -0,0 +1,57 @@
+# Spack %%spack_version%% Container Image
+
+Spack is a package manager for supercomputers. It provides build recipies
+for > 6k software components and allows to build entire HPC application
+stacks with little to no prerequisits installed.
+
+This container may be used as a build container when creating a `Dockerfile`
+ or `appainter.def` file using `spack containerize`. It can be used to  run
+ spack commands directly as well. This may require to bind-mount local
+directories into the container.
+
+## How to use the image
+
+This image may be used to build and containerize application stacks using
+Spack. The stack will be installed in a base container such as SLE BCI Base.
+To build a containerized application stack, simply create a file `spack.yaml`
+in an empty directory:
+```
+spack:
+  specs:
+  - <application spec>
+
+  container:
+    format: <container_format>
+    images:
+      build: "{{ image.reference }}"
+      final: "registry.suse.com/bci/bci-base:latest"
+```
+The 'application spec' consists of the application name optionally augmented
+with additional build specifications (for details consult the
+[Spack documentation](https://spack.readthedocs.io/en/latest/)).
+ The 'container_format' may be either ``docker`` for a Docker/podman container
+ or ``singularity`` for a Singularity/Apptainer container.
```

Comment:
```suggestion
The 'application spec' consists of the application name with additional
optional build specifications (for details consult the
[Spack documentation](https://spack.readthedocs.io/en/latest/)).
The 'container_format' can be either `docker` for a docker/OCI container image
or ``singularity`` for a Singularity/Apptainer container image.
```

Reply:
Also reworded. Also used 'OCI' instead of 'podman' and added 'image' to container.